spec/ruby/optional/capi/thread_spec.rb

```diff
@@ -84,5 +84,9 @@ def call_capi_rb_thread_wakeup
 
       lambda { thr.join }.should raise_error(NotImplementedError)
     end
+
+    it "sets the thread's group" do
+      @t.rb_thread_create(lambda {}, nil).group.should_not be_nil
```

Comment:
Is it possible here to test for the type that it _does_ return, instead of merely testing if it's not nil?